src/intentLogic/getOpinion.js

```diff
@@ -0,0 +1,31 @@
+const get = require("simple-get-promise").get;
+const asJson = require("simple-get-promise").asJson;
+
+const helpers = require('../helpers');
+const speech = require('../speech').speech;
+const sound = require('../speech').sound;
+
+const config = require("../../tmp/config.json");
+const CAPI_API_KEY = config.capi_key;
+
+module.exports = function (search_term) {
+	this.event.session.attributes.lastIntent = 'GetOpinion';
+
+	var capi_filter = 'show-fields=standfirst,byline,headline&show-blocks=all&tag=commentisfree/commentisfree';
+	var capi_query = helpers.capi_query('search', capi_filter, search_term);
+
+	get(capi_query)
+		.then(asJson)
+		.then(json => {
+			if (json.response.results && json.response.results.length > 1) {
+				var opinion_speech = speech.acknowledgement + speech.opinions.latest;
+				opinion_speech += json.response.results[0].fields.headline + ' by ' + json.response.results[0].fields.byline + '. ' + json.response.results[0].blocks.body[0].bodyTextSummary;
```

Comment:
The view that people care about whom has written an article is still one I don't fully believe in.